src/modules/RootPart.rb

```diff
@@ -1714,6 +1716,16 @@ def create_backup
       end
     end
 
+    def pre_inject_files
```

Comment:
Term "inject" means adding from outside. In this case we are just moving the files inside `/mnt`. E.g. the `inject_intsys_files` method copies from inst-sys (`/`) into `/mnt`. So I'd prefer a different method name here.